### PR TITLE
LPS-124366 if not null and previously version has exception, put required

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
It only shows when the file is the extension is not null and previously version has exception, but I cannot know how to check the actual file if included

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by ambrinchaudhary — APPROVED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp [hunk 1]
```diff
@@ -317,7 +317,7 @@ renderResponse.setTitle(headerTitle);
 
 					<aui:input label="title" name="title" />
 
-					<aui:input label="file-name" name="fileName" type='<%= dlEditFileEntryDisplayContext.isFileNameVisible() ? "text" : "hidden" %>' />
+					<aui:input label="file-name" name="fileName" required="<%= fileVersion!= null && Validator.isNotNull(fileVersion.getExtension()) %>" type='<%= dlEditFileEntryDisplayContext.isFileNameVisible() ? "text" : "hidden" %>' />
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:320]: @AliciaGarciaGarcia Can you surround `fileVersion != null` in parens and add a space between the variable and the operator? It should be `<%= (fileVersion != null) && ...`.

Comment (adolfopa) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp:320]: In general, all subexpressions that involve binary operators should always be surrounded by parens, and binary operators need to be surrounded by spaces.

**Review by adolfopa — CHANGES_REQUESTED** [review]
There are some SF issues that need to be fixed before forwarding. See my inline comments.

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp]: Thanks for pointing at it !!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by boton — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**